packages/teams-js/src/public/runtime.ts

```diff
@@ -148,6 +148,7 @@ interface IRuntimeV2 extends IBaseRuntime {
 interface IRuntimeV3 extends IBaseRuntime {
   readonly apiVersion: 3;
   readonly hostVersionsInfo?: HostVersionsInfo;
+  readonly isNAAChannelRecommended?: boolean;
```

Comment:
this is mostly related to my other question on isChannelRecommended, so we are introducing a new param in RunTime for supportivity check for NAA? Is that by design that we are not leveraging existing "Suppports"check for NAA?